src/models/Pet.js

```diff
@@ -0,0 +1,83 @@
+const mongoose = require('mongoose');
+
+const PetSchema = new mongoose.Schema(
+  {
+    petName: {
+      type: String,
+      required: [true, 'Please provide pet name'],
+      maxlength: [100, 'Name can not be more than 100 characters'],
+    },
+    petType: {
+      type: String,
+      enum: ['Cat', 'Dog', 'Other'],
+      default: 'Cat',
+      required: [true, 'Please provide type'],
+    },
+    date: {
+      type: Date,
+      default: Date.now,
+    },
+    description: {
+      type: String,
+      required: [true, 'Please provide product description'],
+      maxlength: [2000, 'Description can not be more than 2000 characters'],
+    },
+    image: {
+      type: String,
+      default: '/uploads/example.jpeg',
+    },
+    breed: {
+      type: String,
+      enum: [
```

Comment:
I do not think it is a good idea to make a pet breed an enum. Countless number of breeds exist and I do not think you should limit those. Also, I am not sure the ones you chose for enums are actual 'breeds'.